File: app/src/main/java/org/dhis2/usescases/jira/JiraComponent.java
package org.dhis2.usescases.jira;

import org.dhis2.data.dagger.PerFragment;

import dagger.Subcomponent;

/**
 * QUADRAM. Created by ppajuelo on 24/05/2018.
 */
@PerFragment
@Subcomponent(modules = JiraModule.class)
public interface JiraComponent {
    void inject(JiraFragment programFragment);
}

File: app/src/main/java/org/dhis2/usescases/jira/JiraComponent.kt
package org.dhis2.usescases.jira

import dagger.Subcomponent
import org.dhis2.data.dagger.PerFragment

@PerFragment
@Subcomponent(modules = [JiraModule::class])
interface JiraComponent {
    fun inject(jiraFragment: JiraFragment)
}
